shipyard_blueprints/databricks/shipyard_databricks/databricks.py

```diff
@@ -1,5 +1,5 @@
+import databricks
 from shipyard_templates import CloudStorage
-import databricks_cli
 from databricks_cli.sdk.api_client import ApiClient
```

Comment:
Not sure if ` ApiClient` is being used. Looks like its part of commented out code. Also may want to cleanup the commented code while we're making changes to this

Reply:
done